llvm/unittests/IR/InstructionsTest.cpp

```diff
@@ -406,8 +406,8 @@ TEST(InstructionsTest, CastCAPI) {
   Type *FloatTy = Type::getFloatTy(C);
   Type *DoubleTy = Type::getDoubleTy(C);
 
-  Type *Int8PtrTy = PointerType::get(Int8Ty, 0);
-  Type *Int32PtrTy = PointerType::get(Int32Ty, 0);
+  Type *Int8PtrTy = PointerType::get(C, 0);
+  Type *Int32PtrTy = PointerType::get(C, 0);
```

Comment:
We should combine these into one PtrTy and then drop any tests that became redundant as a result.

Reply:
I also took care of some the same issue in the test above and below.